text/0000-a11y-routing.md

```diff
@@ -0,0 +1,91 @@
+
+> If the corresponding HTML node of the dynamic content can be focused programmatically, screen reading software will start speaking the textual content of that node. Focusing the corresponding HTML node of the dynamic content can be considered guided focus management. Not only will it facilitate the announcement of the textual content of the focused HTML node, but it will also serve as a guided context switch. Any subsequent “tab” key press will focus the next focusable element within/after this context. 
+
+The tedious, error-prone task of keeping track of HTML nodes that can/could/would/should have focus is something that Ember could do for you. 
+
+A comparable from another JS Ecosystem: [Reach Router](https://reach.tech/router) is available for React apps. 
+
+## How we teach this
+
+For phase one, the guides should include information about skip links, since the application author will need to add those in themselves (just as they would for a static site). For phase two, we would explain that skip links are no longer needed because we have integrated that functionality in a more machine intelligent way. 
+
+### Skip Links
+A skip link is an internal link at the beginning of a hypertext document that permits users to skip navigational material and quickly access the document's main content. Skip links are particularly useful for users who access a document with screen readers and users who rely on keyboards.
+
+Directly inside the `body` element, place a skip link like this: 
+
+    <a href="#maincontent">Skip to main content</a> 
+
+This allows the user with assistive technology to skip directly to the main content and is useful for not needing to repeatedly navigate through a navbar, for example.  
+
+
+It's possible that this solution is a [spherical cow](https://en.wikipedia.org/wiki/Spherical_cow), and the complexity of route transitions in a framework deserves an accessibility solution that is is equally complex. 
+
+It's possible that enterprise Ember users have already implemented their own solution, since governments in many countries require this by law. An out of the box solution would need to be well-advertised and documented to avoid any potential conflicts for users. 
+
+## Alternatives
+1. We could try to implement something like Apple’s first responder pattern: https://developer.apple.com/documentation/uikit/uiresponder/1621113-becomefirstresponder 
+2. Wait for a SPA solution to be implemented in assistive technology. I would note that this option seems less than ideal, since this specific issue has been reported to the different assistive technologies available, and has been an on-going issue for many years now. 
+
+## Unresolved Questions
+- how would we handle loading states?
+- how does/could/would this fit in with the router helpers work that Chad did?
```

Comment:
This is likely just an implementation detail and doesn't really need to be addressed in this RFC. This would clean up maybe a dozen lines of code in `ember-a11y` except that I want for `ember-a11y` to remain backwards compatible, so it won't adopt those until the old APIs are removed.

Given that, ideally any solution would _not_ leverage those helpers unless it had to.